pyclang/idf_extension.py

```diff
@@ -1,8 +1,18 @@
 import os.path
 import sys
-
+import shutil
 from pyclang import Runner
 
+def check_esp_clang_tidy():
+    clang_path = shutil.which("clang-tidy")
+    # xtensa is used only till ESP-IDF v5.0
+    if not clang_path or not any(sub in clang_path for sub in ('esp-clang', 'xtensa-esp32-elf-clang')):
+        raise SystemExit(
+            'ERROR: Espressif Clang was not found on the system.\n\n'
+            'For installation instructions, generate and open the documentation with:\n'
```

Comment:
I'm sorry if I mislead you but this command won't generate the documentation. It will either print to the console or open it in the default browser (docs.espressif.com).